scripts/sysman/fabric.yml

```diff
@@ -185,6 +185,26 @@ members:
       name: "txCounter"
       desc: "[out] Monotonic counter for the number of bytes transmitted (sum of all lanes). This includes all protocol overhead, not only the GPU traffic."
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Fabric Port Error Counters"
+class: $sFabricPort
+name: $s_fabric_port_error_counters_t
+base: $s_structure_type_t
```

Comment:
`$s_structure_type_t` is an enum and cannot be the base type. As the members of this struct are properties of the fabric port error counters `$s_base_properties_t` would be the appropriate base type and would provide the extendibility of having an `stype` and `pNext`.

Reply:
Fixed.